source/getting-started/quickstart.md

```diff
@@ -0,0 +1,262 @@
+
+Now let's tell Ember how to turn that array of strings into HTML. Open
+the `scientists` template and add some Handlebars code to loop through the
+array and print it:
+
+```app/templates/scientists.hbs{+3,+4,+5,+6,+7}
+<h2>List of Scientists</h2>
+
+<ul>
+  {{#each model as |scientist|}}
+    <li>{{scientist}}</li>
+  {{/each}}
+<ul>
+```
+
+Here, we use the `each` helper to loop over each item in the array we
+provided from the `model()` hook and print it inside an `<li>` element.
+
+## Create a UI Component
+
+As your application grows and you notice you are sharing UI elements
+between multiple pages (or using them multiple times on the same page),
+Ember makes it easy to refactor your templates into reusable components.
+
+Let's create a `people-list` component that we can use
+in multiple places to show a list of people.
+
+As usual, there's a generator that makes this easy for us. Make a new
+component by typing:
+
+```sh
+ember generate component people-list
+```
+
+Copy and paste the `scientists` template into the `people-list`
+component's template and edit it to look as follows:
+
+```app/templates/components/people-list.hbs
+<h2>{{title}}</h2>
+
+<ul>
+  {{#each people as |person|}}
+    <li>{{person}}</li>
+  {{/each}}
+<ul>
+```
+
+Note that we've changed the title from a hardcoded string ("List of
+Scientists") to a dynamic property (`{{title}}`). We've also renamed
+`scientist` to the more-generic `person`, decreasing the coupling of our
+component to where it's used.
+
+Save this template and switch back to the `scientists` template. Replace all
+our old code with our new componentized version. Components look like
+HTML tags but instead of using angle brackets (`<tag>`) they use double
+curly braces (`{{component}}`). We're going to tell our component:
+
+1. What title to use, via the `title` attribute.
+2. What array of people to use, via the `people` attribute. We'll
+   provide this route's `model` as the list of people.
+
+```app/templates/people.hbs{-1,-2,-3,-4,-5,-6,-7,+8}
+<h2>List of Scientists</h2>
+
+<ul>
+  {{#each model as |scientist|}}
+    <li>{{scientist}}</li>
+  {{/each}}
+<ul>
+{{people-list title="List of Scientists" people=model}}
+```
+
+Reload the page in your browser and you should see that the UI looks
+identical. The only difference is that now we've componentized our list
+into a version that's more reusable and more maintainable.
+
+You can see this in action if you create a new route that shows a
+different list of people. As an exercise for the reader, you may try to
+create a `programmers` route that shows a list of famous programmers. By
+re-using the `people-list` component, you can do it in almost no code at
+all.
+
+## Building For Production
+
+Now that we've written our application and verified that it works in
+development, it's time to get it ready to deploy to our users. To do so,
+run the following command:
+
+```sh
+ember build
```

Comment:
The next paragraph does say that it packages everything into an optimized bundle, so it should really include that (or the friendlier `-prod`)